src/main/java/com/oltpbenchmark/util/SQLUtil.java

```diff
@@ -512,14 +512,6 @@ private static AbstractCatalog getCatalogDirect(DatabaseType databaseType, Conne
 
         Map<String, Table> tables = new HashMap<>();
 
-        List<String> excludedColumns = new ArrayList<>();
-
-        if (databaseType.equals(DatabaseType.COCKROACHDB)) {
-            // cockroachdb has a hidden column called "ROWID" that should not be directly used via the catalog
-            excludedColumns.add("ROWID");
```

Comment:
Why didn't you just add `rowid` here?

Reply:
This is because the hidden columns to be excluded here must be determined using the `startsWith()` method, not the `equals()` method applied to each element of `excludedColumns`. `excludeColumns` is only used to exclude a hidden column `rowid` for CockroachDB.

Reply:
For example, the DDL below generates a hidden column `crdb_internal_ts_shard_20`.

```sql
CREATE TABLE products (
    ts DECIMAL PRIMARY KEY USING HASH WITH (bucket_count = 20),
    product_id INT8
    );
```

A suffix of a hidden column name is determined based on the primary key name and bucket size specified in the DDL. We can not exclude those columns by simply applying `equals()`.

Reply:
Actually, what if we create a list of compiled regular expressions and then have it check that instead having separate logic for `equals` and `startsWith`.